Assets/Treasure/TDK/Runtime/Connect/TDK.Connect.cs

```diff
@@ -168,6 +173,22 @@ public async Task ConnectSocial(SocialAuthProvider provider)
             await ConnectWallet(ecosystemWalletOptions);
         }
 
+        // TODO stop requiring WalletConnectModal to be in the scene
```

Comment:
Not sure how viable this is without modifying the plugin's code. Perhaps it can be done in a follow up release if some partner needs it